relay: batch sent rows (#9922)
tarantool/tarantool: changelogs/unreleased/gh-10161-relay-speedup.md

## bugfix/replication

> * Significantly improved the replication performance by batching rows to be sent
>   before the dispatch. Batch size in bytes may be controlled by the new tweak
>   option `relay_stream_flush_size` (default 16 kb) (gh-10161).

Reviewer: ```suggestion
* Significantly improved replication performance by batching rows to be sent
  before dispatch. Batch size in bytes may be controlled by a new configuration
  option `relay_stream_flush_size` (default is 16 kb) (gh-10161).
```
